lib/app/services/api.dart

```diff
@@ -1,8 +1,12 @@
 import 'dart:convert';
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
```

Comment:
Is this actually needed? For something like `Color`, the `flutter/foundation.dart` or `dart:ui` package should be enough (not sure which of those).